### PR TITLE
Changelog for Clippy 1.83 🤖

### PR DESCRIPTION
**Pull request by xFrednet**
This is in honor of `@bors` who is no longer used in this repo. I've been saving this poem for a special PR, but it just seems fitting to use it now:

Bors the bot,
Handsome and strong!
Will you go with me,
To prom?

---

### The cat of this release is Abu nominated by @jdonszelmann :

The cats of this Clippy release

Cats for the next release can be nominated in the comments :D

---

changelog: none

### CODE REVIEW

**Comment by jdonszelmann** [comment]
hi Abu!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,56 @@ document.
...
+
+### Moves and Deprecations
+
+* Moved [`ref_option`] to `pedantic` (From `nursery`)
+  [#13469](https://github.com/rust-lang/rust-clippy/pull/13469)
+* Moved [`manual_c_str_literals`] to `complexity` (From `pedantic` now warn-by-default)
+  [#13263](https://github.com/rust-lang/rust-clippy/pull/13263)
+* Moved [`empty_line_after_doc_comments`] to `suspicious` (From `nursery` now warn-by-default)
+  [#13091](https://github.com/rust-lang/rust-clippy/pull/13091)
+* Moved [`empty_line_after_outer_attr`] to `suspicious` (From `nursery` now warn-by-default)
```
Comment (flip1995) [inline CHANGELOG.md:48]:
```suggestion
* Moved [`manual_c_str_literals`] to `complexity` (From `pedantic`, now warn-by-default)
  [#13263](https://github.com/rust-lang/rust-clippy/pull/13263)
* Moved [`empty_line_after_doc_comments`] to `suspicious` (From `nursery`, now warn-by-default)
  [#13091](https://github.com/rust-lang/rust-clippy/pull/13091)
* Moved [`empty_line_after_outer_attr`] to `suspicious` (From `nursery`, now warn-by-default)
```

**Review by flip1995 — APPROVED** [review]
LGTM, thanks! Just a NIT.

**Comment by blyxyas** [comment]
Speaking of that, we should totally organize the cat vote for 1.84. I think that the community has refreshed from the last cat vote we did. What does the cat council think?

**Comment by jdonszelmann** [comment]
Oh do you need an extra cat? one sec lemme get lizzy

**Comment by jdonszelmann** [comment]
lizzy

**Comment by jdonszelmann** [comment]
on the heated bathroom floor, of course...

**Comment by xFrednet** [comment]
The big question is if we get enough cats to have a proper vote. If so, go ahead, cat pics are always nice!

**Pull request merged**